src/test/scala/temple/builder/project/ProjectBuilderTestData.scala

```diff
@@ -93,6 +96,184 @@ object ProjectBuilderTestData {
       |EXPOSE 1024
       |""".stripMargin
 
+  val kongFiles: Map[File, FileContent] = Map(
+    File("kube/kong", "kong-deployment.yaml")    -> FileUtils.readResources("kong/kong-deployment.yaml"),
+    File("kube/kong", "kong-service.yaml")       -> FileUtils.readResources("kong/kong-service.yaml"),
+    File("kube/kong", "kong-db-deployment.yaml") -> FileUtils.readResources("kong/kong-db-deployment.yaml"),
+    File("kube/kong", "kong-db-service.yaml")    -> FileUtils.readResources("kong/kong-db-service.yaml"),
+    File("kube/kong", "kong-migration-job.yaml") -> FileUtils.readResources("kong/kong-migration-job.yaml"),
+  )
+
+  val simpleTemplefileKubeScripts: Map[File, FileContent] = Map(
+      File("kube/TempleUser", "deployment.yaml") ->
+      """apiVersion: apps/v1
+        |kind: Deployment
+        |metadata:
+        |  name: TempleUser
+        |  labels:
+        |    app: TempleUser
+        |    kind: service
+        |spec:
+        |  replicas: 1
+        |  selector:
+        |    matchLabels:
+        |      app: TempleUser
+        |      kind: service
+        |  template:
```

Comment:
Yeah this is quite hard to follow - can we break it into separate variables for each file's content